src/latex/generator.ml

```diff
@@ -310,9 +316,12 @@ let rec block ~in_source (l : Block.t) =
   let one (t : Block.one) =
     match t.desc with
     | Inline i -> inline ~verbatim:false ~in_source:false i
-    | Paragraph i ->
-        inline ~in_source:false ~verbatim:false i
+    | Paragraph (Some p_style, i) ->
+        [(Paragraph_style (p_style, inline ~in_source:false ~verbatim:false i))]
         @ if in_source then [] else [ Break Paragraph ]
+    | Paragraph (None, i) ->
+      [(Paragraph_style (`Left, inline ~in_source:false ~verbatim:false i))] (*FIXME: you may want to handle me well*)
```

Comment:
```suggestion
      inline ~in_source:false ~verbatim:false i
```
This should be calling the previous code.
Also, I think the paragraph logic shouldn't be duplicated, I suggest moving the optional logic to an other function:
```ocaml
  maybe_paragraph_style p_style (inline ~in_source:false ~verbatim:false i)
  @ ...
```